### PR TITLE
Add Members 01-15-2021

### LINKED ISSUE #2421: REQUEST: New membership for n3wscott
### GitHub Username
@n3wscott

### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @thejoycekung
- @palnabarun

### List of contributions to the Kubernetes project

CI Signal shadow for Kubernetes 1.21

### LINKED ISSUE #2422: REQUEST: New membership for s278gupt
### GitHub Username
@s278gupt

### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @thejoycekung
- @palnabarun 

### List of contributions to the Kubernetes project
CI Signal shadow for Kubernetes 1.21

### LINKED ISSUE #2424: REQUEST: New membership for desponda
### GitHub Username
@desponda

### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @erismaster 
- @jeremyrickard 

### List of contributions to the Kubernetes project
- Bug Triage Shadow for Kubernetes 1.21

### LINKED ISSUE #2426: REQUEST: New membership for ams0
### GitHub Username
@ams0

### Organization you are requesting membership in
@kubernetes

### Requirements
- [X] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [X] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [X] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [X] I am actively contributing to 1 or more Kubernetes subprojects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @erismaster 
- @onlydole 

### List of contributions to the Kubernetes project
- Bug Triage Shadow for Kubernetes 1.21

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Fixes #2426 
Fixes #2424 
Fixes #2422 
Fixes #2421 

/hold until EOD for any more requests

cc: @ameukam @palnabarun

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -905,6 +908,7 @@ members:
 - ruiwen-zhao
 - runcom
 - russellb
+- s278gupt
```
Comment (palnabarun) [inline config/kubernetes/org.yaml:911]: Maybe this is failing the test. Most probably numbers come after punctuations in the lexical order. TIL if that's true,

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: config/kubernetes/org.yaml (hunk 1 above)
Comment (savitharaghunathan) [reply config/kubernetes/org.yaml:911]: I was thinking the same. Figured it would throw an error if the sorting was wrong. Fix is on the way

Comment (savitharaghunathan) [reply config/kubernetes/org.yaml]: Done ✅

Comment (palnabarun) [reply config/kubernetes/org.yaml]: Cool! It was actually that. New thing learned.

Comment (savitharaghunathan) [reply config/kubernetes/org.yaml]: I am never going to remember it. It is going to be one of those things where I always do the wrong thing first 😆

**Comment by palnabarun** [comment]
/lgtm

(in case if the changes remain the same)

**Pull request merged**